sky/cli.py

```diff
@@ -1699,36 +1719,44 @@ def autostop(
     # NOTE(dev): Keep the docstring consistent between the Python API and CLI.
     """Schedule or cancel an autostop or autodown for cluster(s).
 
-    CLUSTERS are the name (or glob pattern) of the clusters to stop.  If both
+    Autostop/autodown will automatically stop or teardown a cluster when it
+    becomes idle for a specified duration.  Idleness means there are no
+    in-progress (pending/running) jobs in a cluster's job queue.
+
+    CLUSTERS are the names (or glob patterns) of the clusters to stop. If both
     CLUSTERS and ``--all`` are supplied, the latter takes precedence.
 
-    If --down is passed, autodown (tear down the cluster; non-restartable) is
-    used, rather than autostop (restartable).
+    Idleness time of a cluster is reset to zero, when any of these happens:
```

Comment:
Curious, where did the prior argument descriptions go? `--down, --idle-minutes, --cancel` etc. Was removing it intended?

Reply:
Yes, it will be shown in the args section when people type `sky autostop -h`. The intention is the help page became too long, didn't fit in 1 page on a laptop. https://github.com/skypilot-org/skypilot/pull/1455#discussion_r1035533777